easybuild/easyconfigs/e/ESPResSo/ESPResSo-4.2.0-foss-2021a-CUDA-11.3.1.eb

```diff
@@ -0,0 +1,54 @@
+easyblock = 'CMakeMake'
+
+name = 'ESPResSo'
+version = '4.2.0'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://espressomd.org/wordpress'
+description = """A software package for performing and analyzing scientific Molecular Dynamics simulations."""
+
+source_urls = ['https://github.com/espressomd/espresso/archive/refs/tags/']
+sources = ['%(version)s.tar.gz']
+checksums = ['e6e17fafa9ba80b46f542dc86e07677a8d641cfd0102ebc4db2f7ee47d5fb6f1']
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+toolchainopts = {'usempi': True, 'pic': True}
+
+builddependencies = [('CMake', '3.20.1')]
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('CUDA', '11.3.1', '', SYSTEM),
+    ('SciPy-bundle', '2021.05'),
+    ('Boost.MPI', '1.76.0'),
+    ('Cython', '0.29.34'),
+    ('HDF5', '1.10.7'),
+    ('Mesa', '21.1.1'),
+    ('GSL', '2.7'),
+    ('IPython', '7.25.0'),
+    ('Pint', '0.20.1'),
+]
+
+configopts = ' -DCMAKE_SKIP_RPATH=OFF -DWITH_TESTS=ON'
```

Comment:
I don't see any test step being run here, missing `runtest = 'test'` perhaps?

Reply:
There are a few tests that are constantly failing in v4.2.0. Jean-Noel (one of the developers) said that these tests will be fixed in a new version (v4.2.1) due to be published this week. I will update this PR with v4.2.1 and will add a proper test step after the release.

Reply:
done